src/Queue/AWSSQS/SqsQueueManager.php

```diff
@@ -413,6 +414,14 @@ private function publishMessage(
             throw SqsClientException::createFromInvalidDelaySeconds($delaySeconds);
         }
 
+        if ($this->eventDispatcher !== null) {
+            $this->eventDispatcher->dispatch(new BeforeMessageSentEvent(
+                $job,
+                $delaySeconds,
+                $prefixedQueueName,
+            ));
+        }
```

Comment:
Not sure how what is the agreement currently on null safe operator, but this seems like good place to use it
```suggestion
        $this->eventDispatcher?->dispatch(
            new BeforeMessageSentEvent(
                $job,
                $delaySeconds,
                $prefixedQueueName,
            )
        );
```